js/main.js

```diff
@@ -0,0 +1,98 @@
+/*
+
+@ingredients = array of Ingredient
+
+*/
+
+var Pantry = function(newIngredients) {
+	this.ingredients = newIngredients;
+};
+
+
+var Ingredient = function(name, type, level, identifier) {
+	this.name = name;
+	this.type = type;
+	this.level = level;
+	this.identifier = identifier;
+	
+};
+
+// These are the items in the pantry
+var myPantry = new Pantry([
+	//Strong Ingredients
+	new Ingredient('Whiskey with', 'strong', 1, 'StrDrink1'),
```

Comment:
I don't think you really need the identifier. I see you are using it in `value=` for the `options` but you could just use `1,2,3` as both `level` & `identifier`. I'm taking it 1 is your strongest etc etc and then `StrDrink#` is keeping straight which drink the user picked. Totally your preference but I think you could choose to use one or the other.